Header header
# Name of task or action
string name
# Whether one-shot action or higher-level task
bool oneshot
# Whether current task should be interrupted
bool interrupt
# Target object
int32 obj_id
# Target location
geometry_msgs/Point location
